exasol_script_languages_container_ci/lib/release.py

```diff
@@ -35,18 +35,29 @@ def release(ctx: click.Context,
     logging.info(f"Running Release build for parameters: {locals()}")
 
     flavor_path = (f"flavors/{flavor}",)
-
+    test_container_folder = "test_container"
     log_path = Path(build_config.DEFAULT_OUTPUT_DIRECTORY) / "jobs" / "logs" / "main.log"
     os.environ[luigi_log_config.LOG_ENV_VARIABLE_NAME] = f"{log_path.absolute()}"
 
-    ci_build(ctx, flavor_path=flavor_path, rebuild=True, build_docker_repository="",
-             commit_sha="", docker_user="", docker_password="")
-    execute_tests(ctx, flavor_path=flavor_path, docker_user=docker_user, docker_password=docker_password)
+    ci_build.build(flavor_path=flavor_path, rebuild=True,
+                   build_docker_repository=None,
+                   commit_sha="",
+                   docker_user=None,
+                   docker_password=None,
+                   test_container_folder=test_container_folder)
+    ci_execute_tests.execute_tests(flavor_path=flavor_path,
+                                   docker_user=docker_user,
+                                   docker_password=docker_password,
+                                   test_container_folder=test_container_folder)
+    ci_security_scan.run_security_scan(flavor_path=flavor_path)
     if not is_dry_run:
-        ci_push(ctx, flavor_path=flavor_path,
-                target_docker_repository=docker_release_repository, target_docker_tag_prefix="",
-                docker_user=docker_user, docker_password=docker_password)
+        ci_push.push(flavor_path=flavor_path,
+                     target_docker_repository=docker_release_repository,
+                     target_docker_tag_prefix="",
+                     docker_user=docker_user,
+                     docker_password=docker_password)
     else:
         logging.info("Skipping push to docker release repository due to dry-run.")
```

Comment:
Replacing the `if` condition with and conditional assignment could be an alternative, which may helps the reader slightly.
```python
def action():
    ci_push(....)
    
def dry_run():
    logging.info(...)
    
action = run if not is_dry_run else dry_run
action()

```

Reply:
To be honest, I think the 'if' is actually easier to understand, but will look if there is a bit better refactoring possible